be/src/exec/vectorized/sorting/sorting.h

```diff
@@ -66,7 +66,16 @@ struct SortDesc {
     int null_first;
 
     SortDesc() = default;
+    SortDesc(bool is_asc, bool inull_first) {
+        sort_order = is_asc ? 1 : -1;
+        null_first = (inull_first ? -1 : 1) * sort_order;
```

Comment:
How about using initialization list to init them? Otherwise, they will be assigned twice.

Reply:
Almost no cost... I choose to make it more clear instead of performance.